contrib/Zipkin/SpanConverter.php

```diff
@@ -51,13 +53,14 @@ public function convert(Span $span)
             ],
             'name' => $span->getSpanName(),
             'timestamp' => (int) ($span->getStartEpochTimestamp() / 1e3), // RealtimeClock in microseconds
-            'duration' => (int) (($span->getEnd() - $span->getStart()) / 1e3), // Diff in microseconds
+            'duration' => (int) (($span->getEnd() - $span->getStart()) / 1e3), // Diff in microseconds,
+            'tags' => [
```

Comment:
It feels itchy that we have to make two separate function calls for the status code and description when they are so tightly coupled together.

Reply:
What do you think we can do about this?

Reply:
I noticed that on Span interface we have the `getStatus` method, but also `getCanonicalStatusCode` and `getStatusDescription` because Span interface extends SpanStatus interface. This is a little bit odd.

Reply:
Yeah that is a bit odd to me too. Not saying this should dn a blocker, just felt unusual to me.  Easy to change down the road should we choose to do so.